frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -709,6 +736,40 @@ export function ContainerInfo(props: ContainerInfoProps) {
     );
   }
 
+  function StatusValue(props: {
+    rows: { name: string; value: string | number; hide?: boolean }[];
+  }) {
+    const { rows } = props;
+
+    const rowsToDisplay = React.useMemo(() => {
+      return rows.filter(({ hide }) => !hide);
+    }, [rows]);
+
+    if (rowsToDisplay.length === 0) {
+      return null;
+    }
+
+    return (
+      <Box>
+        {rowsToDisplay.map(({ name, value }) => {
+          const id = useId('status-value-');
```

Comment:
this violates rule of hooks "Do not call Hooks inside conditions or loops.", if number of rows changes then it will complain that component has different amount of hook calls than in previous render

Reply:
Good catch. I will move it outside.